tools/download-json/main.go

```diff
@@ -33,7 +33,7 @@ type App struct {
 }
 
 func downloadManifest(productId string, result *Manifest) error {
-	manifestUrl := fmt.Sprintf("https://app.scriptoria.io/api/products/%s/files/published/play-listing-manifest", productId)
+	manifestUrl := fmt.Sprintf("https://dev.scriptoria.io/api/products/%s/files/published/play-listing-manifest", productId)
```

Comment:
Don't include this change in your commit

Reply:
I will fix this!